bigtop-packages/src/common/hive/patch16-HIVE-SHELL.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/bin/hive b/bin/hive
+index 1ade51e..f645ef1 100755
+--- a/bin/hive
++++ b/bin/hive
+@@ -293,7 +293,7 @@ if [ "$SKIP_HBASECP" = false ]; then
+     # exclude ZK, PB, and Guava (See HIVE-2055)
+     # depends on HBASE-8438 (hbase-0.94.14+, hbase-0.96.1+) for `hbase mapredcp` command
+     for x in $($HBASE_BIN mapredcp 2>&2 | tr ':' '\n') ; do
+-      if [[ $x == *zookeeper* || $x == *protobuf-java* || $x == *guava* ]] ; then
++      if [[ $x == *zookeeper* || $x == *protobuf-java* || $x == *guava*  || $x == *log* || $x == *slf4j* ]] ; then
```

Comment:
`*log*` looks prone to future bug by matching unintentional files. What is the target jar here?
Same to `*slf4j*`. Are you really sure that all jars matching the pattern should be excluded?